src/vorta/views/repo_add_dialog.py

```diff
@@ -44,6 +48,22 @@ def values(self):
             out['encryption'] = self.encryptionComboBox.currentData()
         return out
 
+    def password_transparency(self):
+        if self.values.get('encryption') != 'none':
+            keyringClass = VortaKeyring.get_keyring().__class__.__name__
+            messages = {
+                'VortaDBKeyring': 'plaintext on disk.\nVorta supports the secure Secret Service API (Linux) and Keychain Access (macOS)',  # noqa
+                'VortaSecretStorageKeyring': 'the Secret Service API',
+                'VortaDarwinKeyring': 'the Secret Service API',
+                'VortaKWallet5Keyring': 'KWallet5',
+                'VortaKWallet4Keyring': 'KWallet4'}
+            # Just in case some other keyring support is added
+            keyringName = messages.get(keyringClass,
+                                       'somewhere that was not anticipated. Please file a bug report on Github')
+            self.passwordLabel.setText('The password will be stored in ' + keyringName)
```

Comment:
same here. use a placeholder.

Reply:
Something like 
`self.passwordLabel.setText(f"The password will be stored in {messages.get(keyringClass, 'somewhere that was not anticipated. Please file a bug report on Github')}")`?